releasenotes/notes/update-ci-visibility-variable-extraction-ade027627cd4bb28.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    Add support for CI provider buddy.works
+upgrade:
+  - |
+    Add `_dd.ci.env_vars` tag for the calculation of the pipeline correlation ID, used to link pipelines and tests executed on jobs within those pipelines.
```

Comment:
If this is an internal implementation detail, we can probably skip it.

Or we can announce it as a separate feature, e.g.:

```
---
features:
 - |
    Add support for CI provider buddy.works
 - |
    CI App: Support properly linking tests to the pipeline that executed them.
```

(my suggested note isn't great, but just an example)